### ISSUE TITLE
React to image service with react templates.

### ISSUE THREAD

**Issue by agentlewis** [Ecosify/COVID-Community-Health-App#60]
## API

Input: string identifying the template we want to generate an image from
Return: an image or a path to the image on the filesystem*

*this is to be determined in implementation.

**Comment by iainkirkpatrick** [comment]
is this for sharing your score as an image @agentlewis ? if so, is that for social media, or other channels aswell?

**Comment by agentlewis** [comment]
@iainkirkpatrick yes primarily for social. Thought I am seeing that there may be different ones for sharing with say a loved one vs publicly.

But yes in short sharing socially
